src/main/java/fr/uga/jandas/Column.java

```diff
@@ -0,0 +1,81 @@
+package fr.uga.jandas;
+
+
+@SuppressWarnings("unchecked")
+public class Column<T>{
+    String label;
+    private String type;
+    T [] elements;
+
+    public Column(String type, int nb_elements){
+        this.type = type;
+        elements = (T[]) new Object[nb_elements];
+    }
+
+    public Column(String type, int nb_elements, T [] arr){
```

Comment:
So the nb_element might reduce the number of element from the original array.
## Example
`arr` is composed of **10** element and i use `Column(Integer, 2, arr)`
now the column will only be composed of the first two elements.

Reply:
I removed this file. I will publish a demonstration file as soon as possible